DXMainClient/DXGUI/Multiplayer/CnCNet/PrivateMessagingWindow.cs

```diff
@@ -18,6 +18,11 @@ namespace DTAClient.DXGUI.Multiplayer.CnCNet
 {
     internal class PrivateMessagingWindow : XNAWindow, ISwitchable
     {
+        // these are used by the "invite to game" feature in the
+        // context menu and are kept up-to-date by the lobby
+        public string inviteGameName;
+        public string inviteGamePassword;
```

Comment:
Have not looked into that part of the code much yet, isn't it possible to directly check if the user is in game lobby?

Reply:
Agree this would be nicer but didn't find a way to do this without making PrivateMessagingWindow aware of CnCNetLobby, which seems wrong to me.  Will have another look into this.

Reply:
Could well be a nice way of doing this with some more investigation but I haven't found one that seems worth the churn.  Happy to look into this more in the future if it's a showstopper.